warehouse/templates/pages/help.html

```diff
@@ -314,13 +314,16 @@ <h3 id="beta">{{ beta() }}</h3>
           <li>We <a href="#feedback">need your feedback</a> to get the site production ready</li>
         </ul>
         <p>
-          While we are in beta, the pypi.org infrastructure cannot yet support all of the API traffic generated by our users running <code>pip install</code>, so don't explicitly point to it in automated production setups. But <a href="https://pyfound.blogspot.com/2018/03/warehouse-all-new-pypi-is-now-in-beta.html">we want you to try it and test it</a>, and we're occasionally redirecting portions of that traffic in load tests we announce <a href="http://status.python.org/">on our status page</a>.
+          We <a href="https://pyfound.blogspot.com/2018/03/warehouse-all-new-pypi-is-now-in-beta.html">entered beta on March 26, 2018</a>. We plan to launch (go from beta to production) and replace pypi.python.org on April 16th, 2018, and shut down pypi.python.org on April 30th, 2018.
```

Comment:
Maybe change "replace pypi.python.org" to "redirect pypi.python.org to pypi.org"?